api/mortarRoute.js

```diff
@@ -247,4 +273,13 @@ module.exports = function(app) {
       });
   });
 
+  // Endpoint for getting a story
+  app.get("/api/story/:id", (req, res) => {
```

Comment:
This directly conflicts with the route defined in `api/storyRoute.js`. Please do the following:
1. compare this route with the older route, and make sure it is returning similar structure (off the cuff, I know you'll have to parse out the `date` from the slug (slugs look like this: `08-11-2016_rust-belt`)
2. hook up an additional route that matches the current `/api/story` route in `api/storyRoute.js` (for the "featured" flag, now that we have explicit ordering in the CMS just always make the top story (order `0`) the featured one)